mapusaurus/respondents/management/commands/load_branch_locations.py

```diff
@@ -14,20 +14,27 @@ def handle(self, *args, **options):
             for branch_location_line in branch_location_reader:
                 record = Branch(
                     year = branch_location_line[0].replace("'", ""),
-                    name = branch_location_line[6],
-                    street = branch_location_line[7] if branch_location_line[7] != '0' else '', 
-                    city = branch_location_line[8],
-                    state = branch_location_line[10],
-                    zipcode = branch_location_line[11],
+                    name = branch_location_line[6].strip().upper(),
+                    street = branch_location_line[7].strip().upper() if branch_location_line[7] != '0' else '',
+                    city = branch_location_line[8].strip().upper(),
+                    state = branch_location_line[10].strip().upper(),
+                    zipcode = branch_location_line[11].strip(),
```

Comment:
`.strip().upper()` can be moved to a `normalize()` function.  This will help if [further processing](https://gist.github.com/j4mie/557354) is necessary